HTML:CSS learning project/style.css

```diff
@@ -7,25 +7,29 @@ header{
   top: 0;
 }
 
-footer {
+.main-footer {
   color:white;
   width: 100%;
   position: fixed;        
   background-color:black;
   padding: 10px 0;
   bottom: 0;
 }
-nav a{
+
+.main-nav a{
   padding: 7px 25px;
   display: inline-block;
 }
+
 .container{
   width: 50%;
   margin: 0 auto;
 }
-img{
+
+.city-image{
   width: 100%;
 }
-body{
+
+.font{
```

Comment:
As I mentioned before, you can use body selector here, however, you need to declare at the top of the CSS file. The unwritten rule is to keep global styles at the top (similar to HTML structure from top to bottom)